Tutorial.md

```diff
@@ -11,477 +12,247 @@ Gist is a paste. A snippet of text. Not unrelated to [GitHub Gists][].  To set s
-                    "created_at": "2014-04-14T02:15:15Z",
-                    "description": "Description of Gist",
-                    "content": "String contents"
-                }
-
-Resource Model is an example representation of the Gist resource. It can be referenced elsewhere at any point where a response (or request) payload is expected. A resource model has *exactly* the same structure as a response payload. In this case it also includes additional description. A Resource Model may not include a response status code.
-
-<a name="refer_model"></a>
-#### Referring the Resource Model
-
-    ### Retrieve a Single Gist [GET]
-    + Response 200
-
-        [Gist][]
-
-With Gist Resource Model in place it is super easy to define an action that retrieves a single gist.
-
-> **NOTE:** The syntax for referencing a resource model is `[<resource identifier>][]`. The model must be referred as "whole" - you can't reuse just a model's body or headers.
-
-## Modifying a Resource
-Let's add an action to our Gist Resource that will allow us to modify its state and another action that deletes a Gist Resource as whole.
-
-    ### Edit a Gist [PATCH]
-    To update a Gist send a JSON with updated value for one or more of the Gist resource attributes. All attributes values (states) from the previous version of this Gist are carried over by default if not included in the hash.
-
-    + Request (application/json)
+```markdown
++ Response 200 (application/json)
 
+        [
             {
-                "content": "Updated file contents"
+                "question": "Favourite programming language?",
+                "published_at": "2014-11-11T08:40:51.620Z",
+                "url": "/questions/1",
+                "choices": [
+                    {
+                        "choice": "Swift",
+                        "url": "/questions/1/choices/1",
+                        "votes": 2048
+                    }, {
+                        "choice": "Python",
+                        "url": "/questions/1/choices/2",
+                        "votes": 1024
+                    }, {
+                        "choice": "Objective-C",
+                        "url": "/questions/1/choices/3",
+                        "votes": 512
+                    }, {
+                        "choice": "Ruby",
+                        "url": "/questions/1/choices/4",
+                        "votes": 256
+                    }
+                ]
             }
+        ]
+```
 
-    + Response 200
+The polls resource has a second action, allowing you to create a new question. Inside this action, there is a description for the structure you would send to the server to create a new question.
 
-        [Gist][]
+```markdown
+### Create a new question [POST]
 
-    ### Delete a Gist [DELETE]
-    + Response 204
+You can create your own question using this action. It takes a JSON dictionary containing a question and a collection of answers in the form of choices.
```

Comment:
Ha, I actually meant object. I am not sure why I typed document, but glad you figured it out.